project/Dependencies.scala

```diff
@@ -15,21 +15,22 @@ object Dependencies {
   lazy val java8CompatVersion = settingKey[String]("The version of scala-java8-compat to use.")
 
   val junitVersion = "4.12"
-  val slf4jVersion = "1.7.25"
+  val slf4jVersion = "1.7.28"
   // check agrona version when updating this
-  val aeronVersion = "1.19.1"
+  val aeronVersion = "1.21.2"
```

Comment:
(important to doublecheck)

Reply:
Not sure about this one (e.g. https://gitter.im/real-logic/Aeron/archives/2018/04/27) - have we looked into this before?

Reply:
the change log is here https://github.com/real-logic/aeron/wiki/Change-Log, and they have good track record so 👍